### ISSUE TITLE
Import-Export: Cryengine Importer not populating on Add-ons tab

### ISSUE THREAD

**Issue by VatoXephan** [Markemp/Cryengine-Importer#21]
I am having an issue with blender not showing the addon for me to enable. I have tried in 3.4 and 3.0, I have both use the install option and also tried extracting the entire zip of the file to the addon folder (C:\Users\username\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons). Using  v2.1.1 of the  Cryengine-Importer

**Comment by Markemp** [comment]
Does it show up in the preferences tab?
![image](https://github.com/Markemp/Cryengine-Importer/assets/12057128/bb760127-da30-4fba-9be2-67fc706e3f83)
Also, make sure you have the Community tab selected.
![image](https://github.com/Markemp/Cryengine-Importer/assets/12057128/d723d6af-60f9-47c9-8f9c-ab1cd2f126ef)

**Comment by ArcAngelEx007** [comment]
Hello. I am having a similar issue. I am able to find the add-on but I get this message. What would you suggest I try next?
<img width="660" alt="Screenshot 2023-10-18 at 7 52 47 am" src="https://github.com/Markemp/Cryengine-Importer/assets/148296309/40a0b5c4-0d3a-4d7e-9159-31db875cdcaa">

**Comment by ArcAngelEx007** [comment]
> Hello. I am having a similar issue. I am able to find the add-on but I get this message. What would you suggest I try next? <img alt="Screenshot 2023-10-18 at 7 52 47 am" width="660" src="https://user-images.githubusercontent.com/148296309/276043597-40a0b5c4-0d3a-4d7e-9159-31db875cdcaa.png">

I should also add that this error message occurs with every file I try to add. It simply will not recognise any of the files. As you can see I am using blender 3.4.

**Comment by VatoXephan** [comment]
Sorry about the late reply, it does not show up on the list to enable at all.



    On Tuesday, October 17, 2023 at 05:59:32 PM MST, ArcAngelEx007 ***@***.***> wrote:  
 
 



Hello. I am having a similar issue. I am able to find the add-on but I get this message. What would you suggest I try next? 


I should also add that this error message occurs with every file I try to add. It simply will not recognise any of the files. As you can see I am using blender 3.4.

**Comment by VatoXephan** [comment]
Sorry about the late reply, it does not show up on the list to enable at all.

![thumbnail-1](https://github.com/Markemp/Cryengine-Importer/assets/140324095/958329ef-be74-47de-9771-22a634973191)

**Comment by VatoXephan** [comment]
> Sorry about the late reply, it does not show up on the list to enable at all.
> 
> ![thumbnail-1](https://user-images.githubusercontent.com/140324095/282320633-958329ef-be74-47de-9771-22a634973191.png)

I was able to resolve this by pointing to the zip file rather than the extracted files when installing from inside blender.

**Comment by ArcAngelEx007** [comment]
Hi there,

Thank you for the reply. I have tried as you suggested and this has not worked either. I still get the same error message.

I am using a macbook and have used this before successfully on mac. I have also tried running this on parallel desktop software. Again to no avail.

Not sure what to make of this as I have done this before but cannot get it to work all of a sudden.

Thank you for your help and would appreciate any extra support or ideas you may be able to give.

From: VatoXephan ***@***.***>
Date: Monday, 13 November 2023 at 2:53 am
To: Markemp/Cryengine-Importer ***@***.***>
Cc: ArcAngelEx007 ***@***.***>, Comment ***@***.***>
Subject: Re: [Markemp/Cryengine-Importer] Import-Export: Cryengine Importer not populating on Add-ons tab (Issue #21)

Sorry about the late reply, it does not show up on the list to enable at all.

[Image removed by sender. thumbnail-1]<https://user-images.githubusercontent.com/140324095/282320633-958329ef-be74-47de-9771-22a634973191.png>

I was able to resolve this by pointing to the zip file rather than the extracted files when installing from inside blender.

**Comment by Markemp** [comment]
Are you clicking the Install button and pointing to the `.zip` file? I just downloaded the 2.1.1 zip file from here and ran through the install prompt.  This is on Blender 3.4, but it should work the same with 3.6.  Let me know if this doesn't work though.

![Add-on install](https://github.com/Markemp/Cryengine-Importer/assets/12057128/08471911-d660-4242-9cd6-11e9b289bcfa)

**Comment by ArcAngelEx007** [comment]
Hi Mate,

That is exactly what I am doing yes, and it still gives the error message. It is quite frustrating as I know this works, I have done it before, but for some reason it will not work now. I will try a delete and fresh install of blender and retry this again tonight. \


[A screenshot of a computer  Description automatically generated]

For reference the full error message.

Thanks though for confirming that I have been doing the right thing. I just cant see what is going wrong. I am following this step by step as outlined on the website and as I have done several times before. I think I just need to wipe clean and try again.


From: Heffay ***@***.***>
Date: Tuesday, 14 November 2023 at 8:39 pm
To: Markemp/Cryengine-Importer ***@***.***>
Cc: ArcAngelEx007 ***@***.***>, Comment ***@***.***>
Subject: Re: [Markemp/Cryengine-Importer] Import-Export: Cryengine Importer not populating on Add-ons tab (Issue #21)

Are you clicking the Install button and pointing to the .zip file? I just downloaded the 2.1.1 zip file from here and ran through the install prompt. This is on Blender 3.4, but it should work the same with 3.6. Let me know if this doesn't work though.

[Add-on install]<https://user-images.githubusercontent.com/12057128/282791799-08471911-d660-4242-9cd6-11e9b289bcfa.gif>

**Comment by ArcAngelEx007** [comment]
Hi Mate,

I have also just realised that I cant get the .zip anymore. The mac keeps extracting it automatically when its downloaded. I compress it but it is not installing. I am wondering if its possible for you to attach the .zip here in the email just to make sure that I get it as a proper.zip

Cheers



From: Heffay ***@***.***>
Date: Tuesday, 14 November 2023 at 8:39 pm
To: Markemp/Cryengine-Importer ***@***.***>
Cc: ArcAngelEx007 ***@***.***>, Comment ***@***.***>
Subject: Re: [Markemp/Cryengine-Importer] Import-Export: Cryengine Importer not populating on Add-ons tab (Issue #21)

Are you clicking the Install button and pointing to the .zip file? I just downloaded the 2.1.1 zip file from here and ran through the install prompt. This is on Blender 3.4, but it should work the same with 3.6. Let me know if this doesn't work though.

[Image removed by sender. Add-on install]<https://user-images.githubusercontent.com/12057128/282791799-08471911-d660-4242-9cd6-11e9b289bcfa.gif>

**Comment by Markemp** [comment]
Let me know if this doesn't work.

**Comment by ArcAngelEx007** [comment]
Hi Mate,

Did you attach the .zip? As I cannot get it any other way at the moment. Damn Mac OS

From: Heffay ***@***.***>
Date: Thursday, 16 November 2023 at 7:36 pm
To: Markemp/Cryengine-Importer ***@***.***>
Cc: ArcAngelEx007 ***@***.***>, Comment ***@***.***>
Subject: Re: [Markemp/Cryengine-Importer] Import-Export: Cryengine Importer not populating on Add-ons tab (Issue #21)
Let me know if this doesn't work.

**Comment by ArcAngelEx007** [comment]
Hi Mate,

So I have borrowed a mates Windows based laptop. It works on that without issues. I was unable to import the mech I wanted as there was an issue with the way I converted it apparently.

As such I think we are good here. It seems that this addon does not work on Mac laptops, even if they are running widows. Sorry for wasting your time. It looks like I will need to get a cheap windows laptop

Cheers for all your help

From: Pete Watson ***@***.***>
Date: Thursday, 16 November 2023 at 7:39 pm
To: Markemp/Cryengine-Importer ***@***.***>, Markemp/Cryengine-Importer ***@***.***>
Cc: Comment ***@***.***>
Subject: Re: [Markemp/Cryengine-Importer] Import-Export: Cryengine Importer not populating on Add-ons tab (Issue #21)
Hi Mate,

Did you attach the .zip? As I cannot get it any other way at the moment. Damn Mac OS

From: Heffay ***@***.***>
Date: Thursday, 16 November 2023 at 7:36 pm
To: Markemp/Cryengine-Importer ***@***.***>
Cc: ArcAngelEx007 ***@***.***>, Comment ***@***.***>
Subject: Re: [Markemp/Cryengine-Importer] Import-Export: Cryengine Importer not populating on Add-ons tab (Issue #21)
Let me know if this doesn't work.

**Comment by Markemp** [comment]
Hmm... I did attach the zip file to the email, but github must have stripped it out.

The other thing you can try is checking the browser or OS settings on your Mac.  There may be a way to have it not auto-extract zipped files.  I don't have a mac to test this on, but see if this helps:  https://docs.themeisle.com/article/86-how-to-fix-safari-automatically-unzip-files

**Comment by ArcAngelEx007** [comment]
Hi Mate,

Thank you for all of that. I managed to get my hand on a second hand windows laptop and have been able to get everything done. It all works.

I honestly think this is just a situation on Mac vs Windows. I am happy that it has worked, even if I had to spend a little cash on a laptop hehe.

Thank you so much for all your help. This is a very cool application and it is seriously making some dreams come true.

Thanks for everything. Ill sign this one as solved and completed.

Cheers
